### ISSUE TITLE
shellcheck error is hard to parse/placed at the wrong line

### ISSUE THREAD

**Issue by dprotaso** [rhysd/actionlint#125]
I notice errors in `run` blocks are hard to understand

ie. below are line numbers in the gutter

```yaml
399 run: |
400 ...
401 bad command
402 ...
```

Will report errors as
```
[actionlint] reported by reviewdog 🐶 
shellcheck reported issue in this script: SC2086:info:2:2: Double quote to prevent globbing and word splitting [shellcheck] 

Raw Output: ./.github/workflows/style.yaml:399:9: shellcheck reported issue in this script: SC2086:info:2:2: Double quote to prevent globbing and word splitting [shellcheck]
```

Github warning is placed at line 399 - but the error is actually at 401.

**Comment by rhysd** [comment]
Please resolve the position in your side.

`399:9:` and `2:2:` are included in the error message. So, the actual position the error occurred was `399+2:9+2` = `401:11`.

I'm understanding that it is not straightforward, but it is hard to accurately calculate the original position considering error message from shellcheck.

For example,

```
- run: "...\nbad command\n..."
```

In this case, the error message would be the same but the actual position on the source file is different.

**Comment by dprotaso** [comment]
Yeah I didn't think of `\n` lines present in the script :(

thanks for the reply
